components/treeview/metadata/treeview.yml

```diff
@@ -144,13 +148,12 @@ examples:
         </li>
       </ul>
 
-  - id: treeview-standalone
-    name: Standalone
+  - id: treeview-detached
+    name: Detached
```

Comment:
Is this just a rename? Does it need to be noted as part of the migration guide?

Reply:
Yep, it was added to the documentation as: 
> * `.spectrum-TreeView-standalone` renamed to `.spectrum-TreeView-detached`
I saw on the Changelog in the Guidelines that this was renamed.